sphinx_rtd_theme/layout.html

```diff
@@ -29,11 +29,19 @@
     <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
   {%- endif %}
   {%- for css in css_files %}
-    {{ css_tag(css) }}
+    {% if css|attr("filename") -%}
+      {{ css_tag(css) }}
+    {%- else -%}
+      <link rel="stylesheet" href="{{ pathto(css, 1)|escape }}" type="text/css" />
+    {%- endif %}
   {%- endfor %}
 
   {%- for cssfile in extra_css_files %}
-    {{ css_tag(cssfile) }}
+    {% if css|attr("filename") -%}
+      {{ css_tag(css) }}
+    {%- else -%}
+      <link rel="stylesheet" href="{{ pathto(css, 1)|escape }}" type="text/css" />
```

Comment:
OR this:
```suggestion
    {% if cssfile|attr("filename") -%}
      {{ css_tag(cssfile) }}
    {%- else -%}
      <link rel="stylesheet" href="{{ pathto(cssfile, 1)|escape }}" type="text/css" />
```

But not both, of course.

Reply:
Also, it looks like `extra_css_files` is not a standard feature of Sphinx, but just something that can be explicitly included in HTML context, like here in @lazka's project:
https://github.com/quodlibet/mutagen/blob/release-1.47.0/docs/conf.py#L45-L49

So, I believe, it will never have a `filename` attribute, so we can further simplify this code block to unconditional:
```html
<link rel="stylesheet" href="{{ pathto(cssfile, 1)|escape }}" type="text/css" />
```